adot/python/src/otel/otel_sdk/otel-instrument

```diff
@@ -2,17 +2,46 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: MIT-0
 
-import os
+from os import environ, system
```

Comment:
By the way, do you expect this script to get more complicated? If only setting variables / interpolating strings, bash script would be more readable and same maintanability.

Reply:
Python have to setup otel component totally by environment variables, sdk does not support combined mode(env variable + code), we don't want to override customer's env variables, so have to add simple operation such as null check, append, etc.